### ISSUE TITLE
Add delete keyboard shortcut

### ISSUE THREAD

**Issue by Vidyadayini** [annotorious/annotorious#193]
I see that createAnnotation and updateAnnotation are called but for some reason deleteAnnotation is not called and so I have to add a keydown listener.

`  annotorious.on('createAnnotation', annotation => handleCreate(annotation)); // works
    annotorious.on('updateAnnotation', annotation => handleUpdate(annotation)); // works
    annotorious.on('deleteAnnotation', annotation => handleDelete(annotation)); // Does not work
    window.addEventListener('keydown', handleDelete); // And so I need to do this`

**Comment by rsimon** [comment]
Hi @Vidyadayini,

you are correct, there is no Delete keyboard shortcut implemented. But I agree, it would be a useful feature addition. I'll put it on the list for the next release.

P.S.: you could also add it yourself, from the outside, without changing the Annotorious code easily. E.g.:

```js
var anno = Annotorious.init({ image: 'myImage' });

window.addEventListener('keydown', function() {
  var selected = anno.getSelected();
  if (selected)
    removeAnnotation(selected);
});
```

**Comment by Vidyadayini** [comment]
Yes. Yes my handleDelete is doing that for Delete key. Thanks for the suggestion and putting this in the list for the next release.

**Comment by vidya-talapady-percipient-ai** [comment]
@rsimon is this also fixed in the 2.7 version ?

**Comment by rsimon** [comment]
Yes, that I believe that's implemented.
